ffdc.sh

```diff
@@ -24,7 +28,7 @@ function close_ffdc {
 }
 
 function fail_unless_bastion {
```

Comment:
This function must be renamed to something like "fail_retrieving_project" since we are no longer using a bastion node. The current name could be misleading.

Reply:
Also, all the references to this function must be changed.